cmd/bnscli/clitests/attach_multisig_id.test.gold

```diff
@@ -0,0 +1,16 @@
+{
+	"multisig": [
+		"MQ==",
```

Comment:
But why are these so short? 1 byte?

If you pull in ints, please treat them as 8-byte bigendian.
If you use hex, fair enough to use it verbatum.